src/app/loadout-builder/LoadoutBuilder.m.scss

```diff
@@ -69,6 +69,21 @@
   }
 }
 
+.undoRedo {
+  display: flex;
+  flex-direction: row;
+  margin: 10px 0;
+
+  > * {
+    flex: 1;
+    margin-right: 4px;
```

Comment:
We have a couple helpers for this sort of thing - `@include horizontal-space-children(4px)`, or `@include flex-gap(4px)` if you want to handle cases where the buttons wrap.